doc/ref/stdlib.html

```diff
@@ -1,12 +1,15 @@
 ---
+# AUTOGENERATED FILE. DO NOT EDIT BY HAND!
 layout: default
 title: Standard Library
 ---
 
 <div class="hgroup">
   <div class="hgroup-inline">
     <div class="panel">
-      <h1 id="standard_library">Standard Library</h1>
```

Comment:
It's missing the id.  Guess we don't need it though.

Reply:
I've added it. The intention in this change is to keep it the same modulo formatting, obvious typos and such.